clients/benchmarks/client.cpp

```diff
@@ -1,5 +1,5 @@
 /* ************************************************************************
- * Copyright 2016 Advanced Micro Devices, Inc.
+ * Copyright 2019 Advanced Micro Devices, Inc.
```

Comment:
I don't know our policy but I thought copyright is supposed to 2016-2019

Reply:
update PR for copywrite format